doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -6,19 +6,19 @@
 
 .. index:: Ceph Block Device; RBD exclusive locks; exclusive-lock
 
-Exclusive locks are a mechanism designed to prevent multiple processes
-from accessing the same Rados Block Device (RBD) in an uncoordinated
-fashion. Exclusive locks are heavily used in virtualization (where
-they prevent VMs from clobbering each others' writes), and also in RBD
-mirroring (where they are a prerequisite for journaling).
+Exclusive locks are mechanisms designed to prevent multiple processes from
+accessing the same Rados Block Device (RBD) in an uncoordinated fashion.
+Exclusive locks are used heavily in virtualization (where they prevent VMs from
+clobbering each other's writes) and in RBD mirroring (where they are a
+prerequisite for journaling).
 
-Exclusive locks are enabled on newly created images by default, unless
-overridden via the ``rbd_default_features`` configuration option or
-the ``--image-feature`` flag for ``rbd create``.
+By default, exclusive locks are enabled on newly created images. This default
+can be overridden via the ``rbd_default_features`` configuration option or the
+``--image-feature`` flag for ``rbd create``.
```

Comment:
Nit: `--image-feature` takes an argument so strictly speaking it's not a flag:
```suggestion
``--image-feature`` option for ``rbd create`` command.
```

Reply:
Also I'd add a `note` here saying that other RBD features such as `object-map` and `fast-diff` depend on `exclusive-lock` feature and that disabling `exclusive-lock` feature will negatively affect performance of some operations.

Reply:
Agreed and altered (in the first instance) and added (in the second instance).